### PR TITLE
Add a react package that can render HTML content properly

### PR DESCRIPTION
**Pull request by PragyaTripathi**
Additionally, fix content wrap issues by removing styling that prevented it.

### CODE REVIEW

File: parijat-frontend/src/routes/content/style.css [hunk 1]
```diff
@@ -9,5 +9,4 @@
 
 .Item__Poem p {
   min-height: 10px;
-  white-space: nowrap;
```
Comment (tux4) [inline parijat-frontend/src/routes/content/style.css]: Why remove the wrapping @PragyaTripathi ?

File: parijat-frontend/src/routes/content/index.js [hunk 2]
```diff
@@ -21,7 +22,7 @@ const Item = ({ result }) => (
 		</h4>
 		<div>
 			{result.text.split('\n').map(paragraph => (
```
Comment (tux4) [inline parijat-frontend/src/routes/content/index.js]: I wonder if we need to `split` and render each lines on our own if we are using `html` to render. Have you tried that? @PragyaTripathi

Comment (PragyaTripathi) [reply parijat-frontend/src/routes/content/index.js]: I did, Prasanna. \n didn't have the same formatting. I like the layout better after putting it in <p> tag.

File: parijat-frontend/src/routes/content/style.css (hunk 1 above)
Comment (PragyaTripathi) [reply parijat-frontend/src/routes/content/style.css]: We now have prose and it wouldn't wrap around the browser width because of this.

**Pull request merged**